src/darwin/Framework/CHIP/zap-generated/MTRBaseClusters.h

```diff
@@ -135,29 +146,60 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
  * Cluster Groups
- *    Attributes and commands for group configuration and manipulation.
+ *
+ * Attributes and commands for group configuration and manipulation.
  */
 @interface MTRBaseClusterGroups : MTRCluster
 
 - (instancetype _Nullable)initWithDevice:(MTRBaseDevice *)device
                               endpointID:(NSNumber *)endpointID
                                    queue:(dispatch_queue_t)queue NS_DESIGNATED_INITIALIZER MTR_NEWLY_AVAILABLE;
 
+/**
+ * Command AddGroup
+ *
+ * Command description for AddGroup
```

Comment:
Do these kind of comments provide any value?

Reply:
Or put it in another way - could we skip the non-human entered strings and only add text that is actually useful?

the `Command Foo` above is also questionable ... I can already see the name `addGroup...` so it is obviously a `Add Group (with parameters)`

Reply:
No, but that's what's in the XML...  The ones that people didn't just punt on and actually put something sane in the XML have useful comments.  And we should fix the XML for the rest.